src/Sparrow.Server/Platform/Pal.cs

```diff
@@ -16,67 +16,17 @@ public static unsafe class Pal
 
         static Pal()
         {
-            var toFilename = LIBRVNPAL;
-            string fromFilename;
-            if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
-            {
-                if (RuntimeInformation.ProcessArchitecture != Architecture.Arm &&
-                    RuntimeInformation.ProcessArchitecture != Architecture.Arm64)
-                {
-                    fromFilename = Environment.Is64BitProcess ? $"{toFilename}.linux.x64.so" : $"{toFilename}.linux.x86.so";
-                    toFilename += ".so";
-                }
-                else
-                {
-                    fromFilename = Environment.Is64BitProcess ? $"{toFilename}.arm.64.so" : $"{toFilename}.arm.32.so";
-                    toFilename += ".so";
-                }
-            }
-            else if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
-            {
-                fromFilename = Environment.Is64BitProcess ? $"{toFilename}.mac.x64.dylib" : $"{toFilename}.mac.x86.dylib";
-                // in mac we are not : `toFilename += ".so";` as DllImport doesn't assume .so nor .dylib by default
-            }
-            else if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
-            {
-                var win7 = PlatformDetails.IsWindows8OrNewer ? "" : "7";
-
-                fromFilename = Environment.Is64BitProcess ? $"{toFilename}.win{win7}.x64.dll" : $"{toFilename}.win{win7}.x86.dll";
-                toFilename += ".dll";
-            }
-            else
-            {
-                throw new NotSupportedException("Not supported platform - no Windows/OSX/Linux is detected ");
-            }
-
-            var retries = 0;
-            while (true)
-            {
-                var e = CopyPalFile(toFilename, fromFilename);
```

Comment:
If I understand this correctly, by using new method we do not need to do any copying. Double checking.

Reply:
Yes, this is no longer needed